pyro/distributions/__init__.py

```diff
@@ -1,3 +1,5 @@
+import torch.distributions.constraints as constraints
```

Comment:
I think this is the only line that conflicts with #2052 . I have added the following to #2052:
```py
from . import constraints, kl, transforms
```